src/routes/authorRouter.ts

```diff
@@ -0,0 +1,13 @@
+import { Router } from 'express';
+import { AuthorController } from '../controller/authorController';
+
+const router = Router();
+
+router.get('/', AuthorController.list);
+router.get('/:id', AuthorController.detail); // Thêm route cho chi tiết
+router.get('/addAuthor', AuthorController.addAuthorForm);
+router.post('/', AuthorController.create);
+router.post('/:id', AuthorController.update);
+router.post('/delete/:id', AuthorController.delete);
```

Comment:
```suggestion
router.put('/:id', AuthorController.update);
router.delele('/:id', AuthorController.delete);
```

Em sửa đường dẫn và method ở view nữa nhé